packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -49,6 +52,7 @@ export interface IDetailsRowProps extends React.Props<DetailsRow> {
   collapseAllVisibility?: CollapseAllVisibility;
   getRowAriaLabel?: (item: any) => string;
   checkButtonAriaLabel?: string;
+  innerZoneFocusible?: boolean;
```

Comment:
Misspelled: should be `isInnerZoneFocusable`.

Reply:
Fixed this.